localdev/seed/api.py

```diff
@@ -203,8 +203,38 @@ def _delete(self, model_cls, data):
         self.seed_result.add_deleted(deleted_type_counts)
         return deleted_count, deleted_type_counts
 
+    def _create_cms_resource_page(self, resource_page):
+        """ Create resource pages in CMS if they don't exists."""
+        existing_obj = ResourcePage.objects.filter(slug=resource_page["slug"]).first()
```

Comment:
why we are using `filter()--first()` here instead of `get` call?  I believe, there should be only and only one resource_page object for each slug. Should we not log exception or handle it in case of `MultipleObjectsReturned` instead of fetching the `first` one from database ?

Reply:
when we use `get` it raises `matching query does not exists` instead of filter where we can simply get object or `None`.

Usage of `get` can lead us to a problem where we don't have an object and this command tries to delete that object.